py/desiutil/test/test_timer.py

```diff
@@ -85,6 +106,80 @@ def test_timer_misuse(self):
         timing_report = t.report()
         self.assertIn('stop', t.timers['blat'].keys())
 
+    def test_parsetime(self):
+        """Test parsing timestamps as int, float, or string"""
+        t0 = parsetime(1600807346)
+        t1 = parsetime(1600807346.0)
+        t2 = parsetime("1600807346")
+        t3 = parsetime("1600807346.0")
+        self.assertAlmostEqual(t0, 1600807346.0)
+        self.assertAlmostEqual(t0, t1)
+        self.assertAlmostEqual(t0, t2)
+        self.assertAlmostEqual(t0, t3)
+
+        timer = Timer()
+        timer.start('blat', starttime=1600807346)
+        timer.stop('blat', stoptime=1600807346+2)
+        self.assertAlmostEqual(timer.timers['blat']['start'], 1600807346.0)
+        self.assertAlmostEqual(timer.timers['blat']['duration'], 2)
+
+    @unittest.skipUnless(dateutil_installed, "dateutil not installed")
```

Comment:
Might want to add a comment that fancy date parsing is currently disabled in the module being tested, but may be reimplemented in the future.

Reply:
comment added about ISO-8601 vs. fancier date parsing in the future.